Packs/Druva/Playbooks/Druva-Ransomware-Response.yml

```diff
@@ -1,491 +1,667 @@
-      brand: ""
+      brand: Druva Ransomware Response
     nexttasks:
       '#default#':
-      - "16"
+      - "9"
       "yes":
-      - "17"
-    scriptarguments:
-      brandname:
-        simple: Druva Ransomware Response
-    results:
-    - brandInstances
+      - "6"
     separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Druva.Resource.resourceID
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
     view: |-
       {
         "position": {
-          "x": 40,
-          "y": 190
+          "x": 1330,
+          "y": 750
         }
       }
     note: false
     timertriggers: []
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
 view: |-
   {
-    "linkLabelsPosition": {
-      "17_18_#default#": 0.9
-    },
+    "linkLabelsPosition": {},
     "paper": {
       "dimensions": {
-        "height": 2105,
-        "width": 1027.5,
-        "x": 40,
-        "y": -40
+        "height": 1745,
+        "width": 1660,
+        "x": 50,
+        "y": -210
       }
     }
   }
 inputs:
-- key: OccuredTime
+- key: UserName
+  value:
+    simple: ${incident.users}
+  required: false
+  description: UserName is used to search userID  of user
+  playbookInputQuery: null
+- key: ResourceName
+  value:
+    simple: ${incident.hostnames}
+  required: false
+  description: ResourceName is used to search resource of type FS,NAS and VMware
+  playbookInputQuery: null
+- key: SiteURL
+  value:
+    simple: ${incident.urls}
+  required: false
+  description: 'SiteURL is used to search sharedrive and sharepoint resources  '
+  playbookInputQuery: null
+- key: DateOfOccurrence
   value:
     complex:
       root: incident
-      accessor: datetimeofthebreach
+      accessor: occurred
       transformers:
-      - operator: Cut
+      - operator: TimeComponents
         args:
-          delimiter:
+          key:
             value:
-              simple: T
-          fields:
+              simple: yyyy-mm-dd
+          time_zone:
             value:
-              simple: "1"
-  required: true
-  description: |-
-    Date when incident occurred. This field will be used as the start date for Druva snapshot quarantine.
-    Format: YYYY-MM-DD
-  playbookInputQuery:
-- key: ComputerName
-  value:
-    complex:
-      root: incident
-      accessor: hostname
-  required: true
-  description: ComputerName is used to search Druva for resourceID
-  playbookInputQuery:
-- key: SHA1
-  value:
-    complex:
-      root: File
-      accessor: SHA1
+              simple: "+09:00"
   required: false
-  description: FIle Hash of ransomware Detected
-  playbookInputQuery:
-outputs:
-- contextPath: Druva.viewedQuarantineRange
-  description: Druva Range Output
-  type: unknown
-tests:
-- Druva-Test
-fromversion: 6.0.0
```

Comment:
Please add back the `fromversion: 6.0.0` key.

Reply:
Change added